### PR TITLE
[Components with more than 1 service] LPS-176579 Refactor DTO Converter components with more than 1 service type from Analytics-Cloud

### PR DESCRIPTION
**Pull request by MrJohn1911**
Related pull: https://github.com/liferay-core-infra/liferay-portal/pull/3932
Ticket: https://issues.liferay.com/browse/LPS-176579

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/analytics/analytics-batch-export-import-impl/src/main/java/com/liferay/analytics/batch/exportimport/internal/dto/v1_0/converter/DXPEntityDTOConverterImpl.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.analytics.dxp.entity.rest.internal.dto.v1_0.converter;
+package com.liferay.analytics.batch.exportimport.internal.dto.v1_0.converter;
```
Comment (rafaprax) [inline modules/apps/analytics/analytics-batch-export-import-impl/src/main/java/com/liferay/analytics/batch/exportimport/internal/dto/v1_0/converter/DXPEntityDTOConverterImpl.java:15]: Hey @MrJohn1911, Could you please add in the commit message the reason why you are moving this class? Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
In this last push, I just edited the commit message of [faa6e6a](https://github.com/liferay-core-infra/liferay-portal/pull/4113/commits/faa6e6af5e3c64700e76d79e89a123319bf60627)

CC: @rafaprax

**Comment by MrJohn1911** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Comment by tinatian** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/133333

**Pull request closed without merging**